test/Event/EventTest.php

```diff
@@ -853,6 +886,35 @@ function (Event $event) use ($locationId) {
             );
     }
 
+    /**
+     * @test
+     */
+    public function it_sets_the_audience_type_to_education_when_setting_a_dummy_education_location()
```

Comment:
```suggestion
    public function it_sets_the_audience_type_to_education_when_setting_a_dummy_education_place()
```

Reply:
Resolving based on conversation in other thread.